src/sphinxcontrib/towncrier/_towncrier.py

```diff
@@ -0,0 +1,23 @@
+"""Towncrier related shims."""
+
+from pathlib import Path
+from typing import Any, Dict
+
+# pylint: disable=import-error,no-name-in-module
+from towncrier._settings import load_config_from_file  # noqa: WPS436
+
+
+# pylint: disable=fixme
```

Comment:
I think global ignores only work at the very top of the module, not after the imports (AFAIR)
```suggestion
```

Reply:
For pylint itself, they work from the line you added until the end or until you enable it.

Reply:
Well, this evidently didn't work as was shown in the CI... I'm pretty sure it applies to the current or the next line usually (per my experience).

Also, I sometimes hit cases when pylint works differently locally vs in CI (this I can't explain, the envs seem equivalent in such cases and I'm pretty sure CIs don't inject user/system level configs either :man_shrugging:).

FWIW this is now fixed in CI and I'm happy with it as it is :)